src/extension/views/doc-source/js/ui.js

```diff
@@ -247,8 +248,9 @@ const load = async () => {
     sendMessage({ fct: 'setMain', params: { html: htmlEditionToSource() } });
   }, 500));
 
-  sampleRUM('sidekick:viewdocsource', {
-    source: tab.url,
+  sampleRUM('click', {
+    source: 'sidekick',
+    target: 'doc-source-opened',
```

Comment:
I think this one would duplicate `context-menu:openViewDocSource` and could be removed, unless there is another way to open this view.

Reply:
True, we are also in this weird place now where `openViewDocSource ` and similar are the only targets where the casing is `camelCase`.

Reply:
Right... I'd be in favor of changing the format of context menu item IDs to use hyphens, at least for their RUM targets. But I think it would be a little odd to change the IDs to past tense just to match the event name standard. Wdyt?